chapter_8/exercise_8_6.py

```diff
@@ -0,0 +1,23 @@
+# exercise 8-6 cities formatter
+# function city country returns a formatted string
+# call at least 3 times and print the output
+def city_country(city: str, country: str) -> str:
+    """Creates formatted city, country string.
+
+    Args:
+        city (str): The city in question.
+        country (str): The country in question.
+
+    Returns:
+        str: The formatted string in the form city, country (title case).
+    """
+    return f'{city.title()}, {country.title()}'
+
+

```

Comment:
I decided to store the return values in variables instead of just immediately printing just for fun. 😄